Pipeline/RunIDZCodeReview/RunCodeReview.groovy

```diff
@@ -89,9 +100,15 @@ else
 	}
 	syslib = syslib.toUnique()
 
+	//If no SYSLIB found and no SYSLIB passed in codereview_syslib and no Property Group file passed, fails and exits
+	if (syslib.size() == 0 && !props.codereview_syslib && !props.codereview_PropertyGroupFile) {
+		println "*** SYSLIB Concatenation is empty and no Property Group file was provided. Exiting..."
+		System.exit(1)
```

Comment:
Rather then exiting with the same RC like on a codereview_maxRC mismatch, maybe exit with rc=2?

We could spend a short section on the README explaining the different return codes:

```
0 - All good
1 - Maximum return code of code inspection exceeded
2 - Wrong configuration
```